src/lib/db-listeners/db-supports-listener.ts

```diff
@@ -23,13 +24,19 @@ export class DBSupportsListener extends BaseListener {
 
   // TODO: Create different listeners for support audiences
   public start() {
+    const { user } = this.db.stores;
     this.supportsRef = this.db.firebase.ref(
-      this.db.firebase.getSupportsPath(this.db.stores.user)
+      this.db.firebase.getSupportsPath(user)
     );
     this.debugLogHandlers("#start", "adding", ["child_changed", "child_added"], this.supportsRef);
     this.supportsRef.on("child_changed", this.onChildChanged = this.handleSupportsUpdate("child_changed"));
     this.supportsRef.on("child_added", this.onChildAdded = this.handleSupportsUpdate("child_added"));
 
+    this.db.firestore.getMulticlassSupportsRef()
+        .where("problem", "==", getProblemPath(this.db.stores))
```

Comment:
`activity-player` uses the offering ID here. Not clear to me a priori which approach is better.

Reply:
The combination of class_hash plus problem will be roughly the same as the offering id. It is possible in the future that the same problem could be assigned to the same class multiple times, but currently that isn't possible. And we haven't added that for 10 years.

The only other reason I can think of making that choice is with student data. In that case we sometimes want to move students between classes, so it would be easier to find all data by a student and class_hash and then switch the class_hash. Instead of having to use a map of old offering ids to new offering ids.  

But in this case the supports are by teachers that should be a problem.

The only other thing about this in general is that it might make sense to follow LTI naming of things. In which case the "class" is really a "context". This is makes some sense sense the assignment of a support could be to any group of users.  So it is up to the thing launch Clue to decide what that context represents.  But I don't think it is worth changing this code. We can easily map "class" to "context" in the future.

Reply:
I went ahead and added some additional LTI-related fields in case they might prove useful down the road.